mpd_oled v1.0.6

supercrab opened the pull request:
Updated readme.md, instal.sh now removes bash script that it creates, plugin moved to system_hardware, node_modules folder should be deleted

volumio commented:
Your node_modules are still in the history.

To send without them:
- Backup the repo you are workign with
- Clone repo again into a new folder
- Copy the changed files ONLY
- Commit and send pull request 

Important is that the node_modules are not committed and then deleted

supercrab commented:
Closing as node_modules are still in the history

supercrab commented:
> Your node_modules are still in the history.
> 
> To send without them:
> 
> * Backup the repo you are workign with
> * Clone repo again into a new folder
> * Copy the changed files ONLY
> * Commit and send pull request
> 
> Important is that the node_modules are not committed and then deleted

Will that still work as the original folder `volumio/volumio-plugins-sources/mpd_oled` still has the `nodes_modules` in it?

volumio commented:
Did not note that... Sorry. 
@balbuze what do you think we shall do to handle this situation in your opinion?

balbuze commented:
humm... I have no node modules in my personnal repo but a zip file of the plugin that contains the working last updated version. When ready, I copy all modified files to my volumio-plugin-sources fork and send he PR from here... Not very handy but that way I can manage other files/folders and don't want to publish (dsp tools files, binaries etc)

supercrab commented:
> humm... I have no node modules in my personnal repo but a zip file of the plugin that contains the working last updated version. When ready, I copy all modified files to my volumio-plugin-sources fork and send he PR from here... Not very handy but that way I can manage other files/folders and don't want to publish (dsp tools files, binaries etc)

@balbuze  I just had a look at your fork to check and it seems to have the `node_modules` folder too.

https://github.com/balbuze/volumio-plugins-sources/tree/master/mpd_oled/

In my latest version of the plugin I have
- improved `install.sh` and `uninstall.sh` 
- added an extra button in the UI to take users to the forum if they have an issue with the plugin
- moved the plugin from `user_interface` to `system_hardware` category (package.json and UIConfig.json)
- updated the language files in i18n folder
- updated readme.md

I haven't changed anything in `index.js` or `config.json`.  

Please let me know how you want me to implement these changes :)

balbuze commented:
The repo you mention is just a fork of volumio-plugin-sources. So if it was in it .. I'm taking about my plugins repo especially the branch alsa _modular (source for FusionDsp)
And the plugin mod_oled I modified is just to show how to use AAMPP.
But I'll have a look into your PR

supercrab commented:
@balbuze OK, Cheers!   Let me know if you need any help with it...

balbuze commented:
where is your PR?

supercrab commented:
> where is your PR and submit?

@balbuze this was my latest PR and the plugin has now been submitted.  When I tried to submit before it told me I hadn't committed the changes even though I had...

balbuze commented:
seems to work but please send a PR without node modules.
I release v1.0.6 as beta
Thanks

supercrab commented:
> seems to work but please send a PR without node modules.
> I release v1.0.6 as beta
> Thanks

Will do now captain 🫡
